forest/src/cli/config.rs

```diff
@@ -23,7 +26,10 @@ impl Default for Config {
             drand_dist_public: DistPublic{coefficients: [hex::decode("82c279cce744450e68de98ee08f9698a01dd38f8e3be3c53f2b840fb9d09ad62a0b6b87981e179e1b14bc9a2d284c985").unwrap(),
                 hex::decode("82d51308ad346c686f81b8094551597d7b963295cbf313401a93df9baf52d5ae98a87745bee70839a4d6e65c342bd15b").unwrap(),
                 hex::decode("94eebfd53f4ba6a3b8304236400a12e73885e5a781509a5c8d41d2e8b476923d8ea6052649b3c17282f596217f96c5de").unwrap(),
-                hex::decode("8dc4231e42b4edf39e86ef1579401692480647918275da767d3e558c520d6375ad953530610fd27daf110187877a65d0").unwrap(),]}
+                hex::decode("8dc4231e42b4edf39e86ef1579401692480647918275da767d3e558c520d6375ad953530610fd27daf110187877a65d0").unwrap(),]},
+            enable_rpc : true,
+            rpc_port: Some("/api".to_string()),
+            rpc_listen: Some("127.0.0.1:8080".to_string())
```

Comment:
```suggestion
            rpc_listen: Some("127.0.0.1:1234".to_string())
```
8080 is probably a bad default port, maybe we should do 1234 since it is Lotus' default and that might be what services plug into by default ?